lib/mixins/state-machine.js

```diff
@@ -87,6 +89,13 @@ module.exports = function StateMachine(Model, settings) {
   const validStates = Model.getEventStates(settings.events)
   const validNames = Model.getEventNames(settings.events)
 
+  // Add the remote methods if enableRest is set
+  if (settings.enableRest) {
+    validNames.forEach(methodName => {
+      Model.remoteMethod(methodName, Model.modelName)
```

Comment:
I think you need to pass the method signature to `remoteMethod`, not just the method name.